### PR TITLE
update the code base to read in hydro evolution from IPGlasma

### PR DESCRIPTION
**Pull request by chunshen1987**
Updates for reading in hydro evolution files from IPGlasma + MUSIC simulations.

### CODE REVIEW

**Review by latessa — APPROVED** [review]
Thanks for this update.  I am adding this PR to the release notes, but I'll wait for Ritoban's review before merging.

**Review by ritobandatta — COMMENTED** [review]
Things appear to be correct and working.

**Pull request merged**
